src/modules/Users.pm

```diff
@@ -4566,10 +4564,13 @@ sub Write {
 		    }
 		}
 
-			# Write authorized keys to user's home (FATE#319471)
-      if (defined($user{"authorized_keys"})) {
-				SSHAuthorizedKeys->write_keys($home, $user{"authorized_keys"});
-      }
+		# Write authorized keys to user's home (FATE#319471)
+		if (defined($user{"authorized_keys"})) {
+		    SSHAuthorizedKeys->write_keys($home, $user{"authorized_keys"});
+		}
+
+		$user{"userPassword"} = "*****"; # Reset before logging
+		y2milestone ("User = ", Dumper(\%user));
```

Comment:
Should we restore back the original password? I mean the code executed later will see the password placeholder instead od the real password. That might be confusing or can cause bugs. I'd rather restore it back, just to be on the safe side and avoid surprises later...